src/Generators/stubs/datatables.stub

```diff
@@ -26,7 +26,7 @@ class DummyClass extends DataTable
      */
     public function query()
     {
-        $query = ModelName::query();
+        $query = ModelName::select($this->getColumns);
```

Comment:
Missing function call here. I suggest you add a `query` before `select` for better IDE support.

```php
$query = Article::query()->select($this->getColumns());
```